Mathlib/RingTheory/Ideal/LocalRing.lean

```diff
@@ -344,6 +344,12 @@ theorem surjective_units_map_of_local_ringHom [CommRing R] [CommRing S] (f : R 
   exact hb
 #align local_ring.surjective_units_map_of_local_ring_hom LocalRing.surjective_units_map_of_local_ringHom
 
+-- see Note [lower instance priority]
+/-- Every ring hom `f : K →+* R` from a field `K` to a nontrivial ring `R` is a local ring hom. -/
+instance (priority := 100) {K R} [Field K] [CommRing R] [Nontrivial R]
```

Comment:
This works for `K` a division ring right?

Reply:
You are right. Thank you!